src/App.css

```diff
@@ -1,51 +1,155 @@
+* {
+	padding: 0;
+	margin: 0;
+	box-sizing: border-box;
+	font-size: calc(10px + 1vmin);
+	text-decoration: none;
+}
+
+:root {
+	--primary-color: #9675b4;
+	--primary-shade-color: #ddd2f0;
+	--secondary-color: #dedede;
+	--note-date-color: #751414;
+
+
+	--black-hl-p5:rgba(0, 0, 0, 0.05);
+	--white-hl-p3:rgba(255, 255, 255, 0.3);
+}
 .App {
-  text-align: center;
+	text-align: center;
 }
 
 .App-logo {
-  height: 40vmin;
-  pointer-events: none;
+	height: 40vmin;
+	pointer-events: none;
 }
 
 @media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+	.App-logo {
+		animation: App-logo-spin infinite 20s linear;
+	}
 }
 
+
+
 .App-header {
-  background-color: #22201c;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  color: white;
+	background-color: #22201c;
+	min-height: 100vh;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	color: white;
 }
 
 .App-link {
-  color: #61dafb;
+	color: #61dafb;
 }
 
 .brand-title {
-  font-size: 3rem;
-  color: #d9d9d9;
+	font-size: 3rem;
+	color: #d9d9d9;
 }
 
 .brand-title span {
-  color: #ffde59;
+	color: #ffde59;
 }
 
 .brand-description {
-  font-size: 1.4rem;
+	font-size: 1.4rem;
 }
 
 .links {
-  display: flex;
+	display: flex;
 }
 
 .links a {
-  color: #d9d9d9;
-  font-size: 1.2rem;
-  margin: 1rem;
+	color: #d9d9d9;
+	font-size: 1.2rem;
+	margin: 1rem;
+}
+
+/* MAin */
+.main-container {
+	display: flex;
+	/* background-color: var(--nav-color); */
+	flex-direction: column;
+	position: relative;
+}
+
+.main-body {
+	display: flex;
+	/* background-color: rgba(0, 0, 0, 0.1); */
+	width: 100%;
+	min-height: 100vh;
+}
+
+.sm-txt {
+	font-size: 0.4em;
+}
+
+.md-txt {
+	font-size: 0.6em;
+}
+
+.lg-txt {
+	font-size: 0.8em;
+}
+
+.xlg-txt {
+	font-size: 1em;
+}
+
+input {
+	font-size: 1em;
+	border-radius: 5px;
+	border: none;
+	padding: 7px;
+	font-weight: 600;
+}
+
+.select-tag {
+	font-size: 0.7em;
+	font-size: 0.7em;
+	border-radius: 5px;
+	border: none;
+	padding: 7px;
+	font-weight: 600;
+}
+
+span {
+	font-size: 1em;
+}
+
+span:hover {
+	color: #539987;
+}
+
+.stretch {
+	width: 100%;
+}
+
+.title-name {
+	color: red;
+}
+
+.page-title {
+	font-size: 1.5em;
+	color: var(--primary-color);
+	font-weight: 800;
+}
+
+.nocontent {
+	display: flex;
+	flex-wrap: wrap;
+	padding: 1em;
+	flex-direction: column;
```

Comment:
write all flex properties together, here it doesn't affect much, but when you have large code block, it will be more readable